.travis.yml

```diff
@@ -158,66 +158,19 @@ script:
-  - python -m pytest -v --durations=10 python/ray/tune/test/tune_server_test.py
-  - python -m pytest -v --durations=10 python/ray/tune/test/ray_trial_executor_test.py
-  - python -m pytest -v --durations=10 python/ray/tune/test/automl_searcher_test.py
+  # `cluster_tests.py` runs on Jenkins, not Travis.
+  - python -m pytest -v --durations=30 --ignore=python/ray/tune/cluster_tests.py python/ray/tune/test
 
   # ray rllib tests
   - python python/ray/rllib/test/test_catalog.py
   - python python/ray/rllib/test/test_filters.py
   - python python/ray/rllib/test/test_optimizers.py
   - python python/ray/rllib/test/test_evaluators.py
 
-  # streaming library tests
-  - python -m pytest -v --durations=10 python/ray/tests/test_batched_queue.py
-  - python -m pytest -v --durations=10 python/ray/tests/test_logical_graph.py
-
+  # ray tests
   # Python3.5+ only. Otherwise we will get `SyntaxError` regardless of how we set the tester.
   - python -c 'import sys;exit(sys.version_info>=(3,5))' || python -m pytest -v --durations=10 python/ray/experimental/test/async_test.py
-
-  # ray tests
-  # TODO(williamma12): We cannot use pytests built-in test discovery because
-  # it causes a lot of the tests to fail on travis' apple builds even though
-  # it runs without issue on an apple build locally.
-  - python -m pytest -v --durations=10 python/ray/tests/test_global_state.py
-  - python -m pytest -v --durations=10 python/ray/tests/test_queue.py
-  - python -m pytest -v --durations=10 python/ray/tests/test_ray_init.py
-  - python -m pytest -v --durations=10 python/ray/tests/test_mini.py
-
-  - python -m pytest -v --durations=10 python/ray/tests/test_basic.py
-  - python -m pytest -v --durations=10 python/ray/tests/test_array.py
-  - python -m pytest -v --durations=10 python/ray/tests/test_actor.py
-  - python -m pytest -v --durations=10 python/ray/tests/test_autoscaler.py
-  - python -m pytest -v --durations=10 python/ray/tests/test_tensorflow.py
-  - python -m pytest -v --durations=10 python/ray/tests/test_failure.py
-  - python -m pytest -v --durations=10 python/ray/tests/test_microbenchmarks.py
-  - python -m pytest -v --durations=10 python/ray/tests/test_stress.py
-  - python -m pytest -v --durations=10 python/ray/tests/test_component_failures.py
-  - python -m pytest -v --durations=10 python/ray/tests/test_multi_node.py
-  - python -m pytest -v --durations=10 python/ray/tests/test_multi_node_2.py
-  - python -m pytest -v --durations=10 python/ray/tests/test_recursion.py
-  - python -m pytest -v --durations=10 python/ray/tests/test_monitors.py
-  - python -m pytest -v --durations=10 python/ray/tests/test_cython.py
-  - python -m pytest -v --durations=10 python/ray/tests/test_credis.py
-  - python -m pytest -v --durations=10 python/ray/tests/test_node_manager.py
-  - python -m pytest -v --durations=10 python/ray/tests/test_signal.py
-  # TODO(yuhguo): object_manager_test.py requires a lot of CPU/memory, and
```

Comment:
Let's try to keep this comment in so we dont forget to move it over to jenkins eventually